src/components/layout.tsx

```diff
@@ -12,11 +12,11 @@ interface Route {
 
 const Layout = (props: React.PropsWithChildren) => {
   return (
-    <div className="h-screen">
-      <div className="flex min-w-[70rem] border rounded-md h-full">
+    <div className="flex w-full justify-center m-4">
```

Comment:
⏳
We should rethink this style. For now, it's acceptable, but I have proposed another solution ✌️